Svg.Tests.Win/SvgDocumentTest.cs

```diff
@@ -156,5 +156,17 @@ public void WhenDocumentOpenWithCss_ElementsAdaptStyle()
             actualFontFamily[0].Should().Be("sans-serif");
             actualFontFamily[1].Should().Be("");
         }
+
+        [Test]
+        public void WhenPathHasClip_CombineBoundsWithPathAndClip()
```

Comment:
again: I cannot understand what is being tested here.
That a clipped path's bounds are calculated by _merging_ the bounds of the clip and the path?

Remember:
`<operation/action>_<precondition>_<expectedoutcome>`